src/prepack-standalone.js

```diff
@@ -51,6 +53,9 @@ export function prepackSources(
     if (!serialized) {
       throw new FatalError("serializer failed");
```

Comment:
This will cause the shutdown call below to not happen.

Reply:
Moved the shutdown above.